docs/howto/openstack/nova/config-drive.md

```diff
@@ -0,0 +1,95 @@
+# Store metadata on a configuration drive
+
+Configuration drive is a read only drive that is attached to an 
+instance during boot. The instance can then mount the drive and
+read files from it. Configuration drives are used as a data source for cloud-init
+and also for passing data to your instances to configure.
+
+# Enable the configuration drive
+
+To enable the configuration drive, you need to pass the parameter `--config-drive`
+with `true` flag to the `openstack server create` command and can also
+specify the user data you want to inject into the server.
+
+## Example shell script
+
+A bash script is a common way to inject user data to an instance.
+The following script will install `tree` command during instance
+creation.
+
+#### **`myscript.sh`**
+```bash
+#!/bin/sh
+
+# Example script to run at boot
+# This installs tree
+sudo apt-get update
+sudo apt install tree
+```
+
+Now, we will pass the `myscript.sh` script using `--user-data` parameter.
+
+```bash
+openstack server create \
+--config-drive true \
+--user-data ./myscript.sh \
+--image ubuntu \
+--flavor m1.small \
+--key-name [KEYPAIR] \
+--nic net-id=[NETWORK_ID] \ 
+guest0
+
+```
+
+This command creates the config drive with volume label `config-2` which is
+attached to the instance during the boot and adds the contents of the file `myscript.sh`
+to the `user_data` file in `openstack/{version}` directory.
+
+To access the config drive, we need to mount the config drive using the following instructions:
```

Comment:
Why is this here?

Reply:
I do not understand this comment. Can you please elaborate?

Reply:
I have removed the bit which explains how to access the config drive.